utils/messageqcpp/inetstreamsocket.cpp

```diff
@@ -575,30 +534,19 @@ void InetStreamSocket::do_write(const ByteStream& msg, uint32_t whichMagic, Stat
   if (msglen == 0)
     return;
 
-  const auto& longStrings = msg.getLongStrings();
   /* buf.fCurOutPtr points to the data to send; ByteStream guarantees that there
      are at least 12 bytes before that for the magic & length fields */
   realBuf = (uint32_t*)msg.buf();
   realBuf -= 3;
   realBuf[0] = magic;
   realBuf[1] = msglen;
-  realBuf[2] = longStrings.size();
```

Comment:
I suppose you can remove this.

Reply:
Then there will be a section of uninitialized memory, which valgrind will criticize me for. 8))